src/Seabug/Contract/Mint.purs

```diff
@@ -54,9 +56,10 @@ mintWithCollection
       { price, lockLockup, lockLockupEnd, authorShare, daoShare }
   ) = do
   owner <- liftedM "Cannot get PaymentPubKeyHash" ownPaymentPubKeyHash
+  ownerStake <- liftedM "Cannot get StakePubKeyHash" ownStakePubKeyHash
   networkId <- getNetworkId
   addr <- liftContractM "Cannot get user address" $
-    payPubKeyHashEnterpriseAddress networkId owner
+    payPubKeyHashBaseAddress networkId owner ownerStake
```

Comment:
Verify this won't be a breaking change, I believe the onchain is coded such that staking addresses aren't permitted right now. We'll need to update that to merge this if this breaks, or move this out to another PR

Reply:
Minting the token purchased in the above transaction was done with this change, so it seems okay?

Reply:
Huh, the source code really looked like it wouldn't allow this, well good to know

Reply:
Staking (enterprise) addresses aren't allowed when changing owner iirc, should work fine for minting. Payment for changing owner has to be sent to address without staking key though.

Reply:
How did the 1 ada cost buy transaction pass then? Was the ownerstake perhaps still Nothing?

Reply:
@samuelWilliams99 I think because the buy contract doesn't use a staking key, it uses the `ownPaymentPubKeyHash` of the buyer

Reply:
oh of course! Sorry thats my bad - I thought this change was in the buy contract, but i see its for minting 😅 
Okay, Im happy with this then, lets merge :D